src/main/webapp/student-ambassador.html

```diff
@@ -70,167 +70,155 @@ <h1>STUDENT <b>AMBASSADOR</b></h1>
-                                                    <button
-                                                            class="btn btn-link w-100 text-dark text-left collapsed panel text-capitalize"
-                                                            type="button"
-                                                            data-toggle="collapse"
-                                                            data-target="#collapseTwo"
-                                                            aria-expanded="false"
-                                                            aria-controls="collapseTwo"
-                                                    >
-                                                        What would I get as an ambassador?
-                                                        <div class="text-primary"><i
-                                                                class="plusMinusSign fa fa-plus float-right"></i></div>
-                                                    </button>
-                                                </h5>
-                                            </div>
-                                            <div
-                                                    id="collapseTwo"
-                                                    class="collapse"
-                                                    aria-labelledby="headingTwo"
-                                                    data-parent="#accordionExample"
-                                            >
-                                                <div class="card-body opacity-8 text-justify">
-                                                    SA’s will be trained and mentored throughout the program via our
-                                                    skills development program formulated with industry experts
-                                                    to give you the in-demand skills employers are seeking for like time
-                                                    management, adaptability and collaboration.
-                                                    The program will arm you to be career ready when you graduate from
-                                                    university. You will also be able to develop your leadership,
-                                                    project management and marketing skills when you deliver SEF
-                                                    programs at campus level.
-                                                </div>
-                                            </div>
+            <h1 class="mb-0 text-onelive font-weight-bold text-center">Frequently Asked Questions</h1>
+            <!--FAQ Accordion-->
+            <div class="accordion">
+                <div class="row">
+                    <div class="col-md-12 ml-auto">
+                        <div class="accordion my-3" id="accordionOne">
```

Comment:
ids should use dashed instead of camelCase

Reply:
done